### DISCUSSION TITLE
Find user from mongo DB using phone number

### DISCUSSION THREAD

**Discussion by codydev-git** [jenssegers/laravel-mongodb#2350] [Q&A]
How are you?
I hope you are doing great.
I need a function to find a user from the database using a phone number.
But in the database, the user's phone number has many formats
+1 (1234) 123-1234
1234567890
(1234) 124 1234
+1234567890
+1 234 567 9012
really many formats...
.....
In this case, How can I find a user with this format "1234567890" phone number?
Thank you

**Comment by rawrstack** [comment]
in this situation, regex is your friend.

**Comment by codydev-git** [comment]
Could you please make a sample?

**Comment by zahidinho** [comment]
If your format is like "1324654897" that. You can simply use this regex :  .*\S\d

**Comment by codydev-git** [comment]
In the database, a user's phone number is "+1 (324) 654-7890"
How can i find the user with "13245647890"?
